holoviews/core/data/grid.py

```diff
@@ -180,7 +180,7 @@ def shape(cls, dataset, gridded=False):
         if gridded:
             return shape
         else:
-            return (np.product(shape), len(dataset.dimensions()))
+            return (np.product(shape,dtype=np.intp), len(dataset.dimensions()))
```

Comment:
Prefer to leave a space between arguments. Same applies below.

Reply:
Sorry, will fix those. 

hv has flake8 whitespace checking turned off on travis. I also couldn't see any instructions for contributors about whitespace (whether it's important, and if so, what the rules are). And when I skimmed the hv code, I saw inconsistent whitespace in general, including spacing between arguments. So I think that makes it a bit tricky for contributors to get whitespace right, unless I've failed to see something?

If you can point me to the rules hv's using, I'll make a PR in docs and/or testing, plus clean up the existing whitespace.

Reply:
We don't have consistent rules mainly (although I think we really should), but that's a whole argument to have. That said I'm very strongly in favor of separating arguments with whitespace and would be in favor of enabling this particular rule as a flake8 check immediately.